debian/xenial/foreman-proxy/control

```diff
@@ -10,7 +10,7 @@ Vcs-Browser: https://github.com/theforeman/smart-proxy
 
 Package: foreman-proxy
 Architecture: all
-Depends: ruby | ruby-interpreter, rake, ruby-sinatra (>= 1.3.3), ruby-rack (>= 1.1.0), ruby-json, ruby-augeas, ruby-bundler-ext, ruby-concurrent (>= 1.0.0), ruby-concurrent (<< 2.0.0)
+Depends: ruby2.3 (>= 2.3.1) | ruby-interpreter, rake, ruby-sinatra (>= 1.3.3), ruby-rack (>= 1.1.0), ruby-json, ruby-augeas, ruby-bundler-ext, ruby-concurrent (>= 1.0.0), ruby-concurrent (<< 2.0.0)
```

Comment:
"ruby" should still get installed, as that's the system-default ruby package, providing (/usr/bin/)ruby, which we use in our scripts. So this should be `ruby | ruby-interpreter, ruby2.3 (>= 2.3.1), rake, ...`